src/main/java/stringcalculator/Calculator.java

```diff
@@ -0,0 +1,70 @@
+package stringcalculator;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.BiFunction;
+import java.util.stream.IntStream;
+
+public class Calculator {
+
+    //  연산식으로 만들기
+    public static String[] trimInputExpression(String targetExpression){
+        return targetExpression.split(" ");
+    }
+
+    //  1. 계산식 공백 체크 (입력값 null)
+    public static boolean inputExpressionIsNotNull(String inputExpression){
+        if (inputExpression == null || inputExpression.isEmpty()){
```

Comment:
```suggestion
        if (Objects.isNull(inputExpression) || inputExpression.isEmpty()){
```
Objects.isNull()을 활용한다면 가독성을 높일 수 있습니다 :)